README.md

```diff
@@ -60,6 +60,8 @@ Good luck.
   - [What is PaaS?](https://azure.microsoft.com/en-us/overview/what-is-paas/)
   - [What is SaaS?](https://azure.microsoft.com/en-us/overview/what-is-saas/)
   - [Creating an Azure account](https://docs.microsoft.com/en-us/dotnet/azure/create-azure-account)
+  - [Types of Database on Azure](https://azure.microsoft.com/en-us/product-categories/databases/)
```

Comment:
"Types of Databases on Azure"

Reply:
Noted on the 's' on database, and will do the changes, thank you for the review